README.md

```diff
@@ -117,6 +117,7 @@ let main =
     model = init (); (* Since model is a set value here, we call our init function to generate that value *)
     update;
     view;
+    renderCallback = (fun _ -> ())
```

Comment:
Hmm, actually thinking this over, perhaps a subscription would make more sense, that way it is fed into the normal even loop processing...  It would have to touch some internals, but as it is defined within tea anyway that seems fine.